app/models/friendship.rb

```diff
@@ -0,0 +1,19 @@
+class Friendship < ApplicationRecord
+  validates :user_id, uniqueness: true, presence: true
+  validates :friend_id, uniqueness: true, presence: true
+
+  belongs_to :user
+  belongs_to :friend, class_name: 'User'
+
+  def self.create_reciprocal_for_ids(user_id, friend_id)
+    Friendship.create(user_id: user_id, friend_id: friend_id)
+    Friendship.create(user_id: friend_id, friend_id: user_id)
```

Comment:
This returns the "friend" that is being added -- I wonder if we care about the return value of this method at any point in time or if we just need it to run.